circle.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
 
 test:
   override:
-    - make
+    - make test
```

Comment:
Hmm.. You are right. We should have the blueprint in helium repo and subtree it in this repo. That way this repo is the last to be updated. This is another thing to do when removing plutonium later.